typescript/scripting-api/index.d.ts

```diff
@@ -49,25 +60,12 @@ declare namespace WoT {
         fragment?: object;
     }
 
-    /** The DiscoveryMethod enumeration represents the discovery type to be used */
-    export enum DiscoveryMethod {
-        /** does not restrict */
-        "any",
-        /** for discovering Things defined in the same Servient */
-        "local",
-        /** for discovery based on a service provided by a Thing Directory */
-        "directory",
-        /** for discovering Things in the same/reachable network by using a supported multicast protocol */
-        "multicast"
-    }
-
     /**
      * The ThingDiscovery object is constructed given a filter and provides the properties and methods
      * controlling the discovery process. 
      */
-    export interface ThingDiscovery {
+    export interface ThingDiscoveryProcess extends AsyncIterable<ThingDescription> {
         filter?: ThingFilter;
-        active: boolean;
         done: boolean;
         error?: Error;
         start(): void;
```

Comment:
There is no _good_ match ..
* stop vs start etc

![image](https://github.com/w3c/wot-scripting-api/assets/11281461/1ced87d0-739f-43ea-a869-7545e8eb940f)

Reply:
Thanks! I think I missed `start` and `next` when adjusted these.

Reply:
Should be resolved by 3be7d2a740dc0b7b343215f40648dfe7baf0659c